src/clj/rems/guide_utils.clj

```diff
@@ -1,7 +1,6 @@
-(ns rems.guide-macros
-  "Utilities for component guide."
+(ns rems.guide-utils
```

Comment:
Why plural? The others are `-util`?

Reply:
Yeah that was a leftover from the previous `guide-utils` namespace. I'll fix it!